### DISCUSSION TITLE
error with checks

### DISCUSSION THREAD

**Discussion by info100t** [amidaware/tacticalrmm#1326] [Q&A]
hi

I have this issue with "checks". i don't understand why :(

![image](https://user-images.githubusercontent.com/105393872/197384972-c7c205fc-ee4e-4d60-a4a9-09e42065e717.png)

I verify my api fqdn and it's ok.. 

how can i solve this ?

thx
romain

**Comment by silversword411** [comment]
Looks like you're trying to use the global keystore. 

You set it up right?
https://docs.tacticalrmm.com/functions/keystore/

**Reply by info100t** [reply]
thanks for your reply.

i tried this but nothing better 

![image](https://user-images.githubusercontent.com/105393872/197405555-e5baf507-cc92-4613-9dfc-ce9e6b4cb759.png)

**Reply by silversword411** [reply]
The name should be ApiURLTactical

You call and use it with the {{}}

See example https://docs.tacticalrmm.com/functions/examples/

**Reply by info100t** [reply]
hi,

So i juste need to create a global keystore like that ?

![image](https://user-images.githubusercontent.com/105393872/197598709-0bae491c-1010-4048-acc3-f3de4bb33af8.png)

**Reply by wh1te909** [reply]
@info100t no, please see below screenshots

![chrome_SJDRYmL66O](https://user-images.githubusercontent.com/7434746/197625705-22f9fafa-0d58-4bb7-9a45-897e0848d9f7.png)
![chrome_zg5eVv9HyK](https://user-images.githubusercontent.com/7434746/197625720-dc2b119a-4cb6-44dc-aa3a-281c328e7557.png)

**Reply by info100t** [reply]
ok i understand... so i modify my configuration but the problem is always here : :-( 

![image](https://user-images.githubusercontent.com/105393872/197628013-8bcf4722-47ed-4d64-8d36-0a501da44701.png)

![image](https://user-images.githubusercontent.com/105393872/197628036-38aeef57-66fa-40f0-b04b-493305117644.png)

![image](https://user-images.githubusercontent.com/105393872/197628074-3e81b848-8f35-49ef-ad11-12d2f2e50fa6.png)

**Reply by silversword411** [reply]
2 brackets, not one
BAD {name}
GOOD {{name}}

Please email help@amidaware.com for paid support if you're still having implementation problems.

**Comment by info100t** [comment]
Yes, but i use bultin tacticalrmm script :(

i don't touch anything.. 

![image](https://user-images.githubusercontent.com/105393872/197634018-f24492c8-3877-4775-b6d0-6bd631cf58ef.png)

**Reply by wh1te909** [reply]
@info100t maybe you have a typo somewhere, check your keystore. see video below


https://user-images.githubusercontent.com/7434746/197638134-fecdfc48-89c3-4aee-9f78-9600bcb54368.mp4

![chrome_VNNkyI8ufc](https://user-images.githubusercontent.com/7434746/197638142-894df665-a9b7-4421-984d-8be0407dd4b1.png)

**Reply by info100t** [reply]
yes !! great ! :)

But now new problem... 📦 

![image](https://user-images.githubusercontent.com/105393872/197863982-36295a33-1032-4d13-b5fe-450e77856076.png)

romain

**Reply by info100t** [reply]
great .... i understand !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
problem solve ! :)

thanks everybody !
